pages/index.js

```diff
@@ -37,6 +38,7 @@ export default function HomePage() {
               title={idea.title}
               hashtags={idea.hashtags}
             />
+            <Link href={`/ideaDetails/${idea.id}`}>See More</Link>
```

Comment:
```suggestion
            <Link href={`/idea/${idea.id}`}>See More</Link>
```
I'd probably suggest a slug like this instead. You'd have to rename your folder as well, etc. Simply because it's more concise and more expected as part of a URL than camelCase. The page itself (the component) should be able to retain the name `IdeaDetails`. As far as I'm aware, this doesn't need to be congruent.

I'd probably suggest the name `IdeaDetailsPage` instead, though, because you're playing spicy roulette with naming variables and pages almost the same. Makes me nervous.

Reply:
We take your suggestion into consideration, but we would want to keep IdeaDetails for clarity!